app/serializers/api/v2/serializable_allocation.rb

```diff
@@ -2,7 +2,7 @@ module API
   module V2
     class SerializableAllocation < JSONAPI::Serializable::Resource
       type "allocations"
-      attributes :number_of_places, :request_type
+      attributes :number_of_places, :request_type, :provider_id
```

Comment:
@asmega - is there any particular reason why you are exposing the `provider_id` here rather than accessing `allocation.provider.id`? The attributes exposed here should only be those that are not accessible via the associations.

Reply:
yep totally. I'm closing this PR as its not needed